Mathlib/MeasureTheory/Measure/RegularityCompacts.lean

```diff
@@ -11,13 +11,14 @@ import Mathlib.Topology.UniformSpace.Cauchy
 /-!
 # Inner regularity of finite measures
 
-The main result of this file is `theorem InnerRegularCompactLTTop`:
+The main result of this file is
+`InnerRegularCompactLTTop_of_pseudoEMetricSpace_completeSpace_secondCountable`:
```

Comment:
I'd be good if somebody double-check that this doc-string change are accurate; I'm only "mostly sure".

Reply:
https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Measure/Regular.html#MeasureTheory.Measure.InnerRegularCompactLTTop exists, although it is in a different file.